extensions/spring-di/deployment/src/main/java/io/quarkus/spring/di/deployment/SpringDIProcessor.java

```diff
@@ -88,6 +89,14 @@ FeatureBuildItem registerFeature() {
         return new FeatureBuildItem(FeatureBuildItem.SPRING_DI);
     }
 
+    /*
+     * In spring-di @Named is a bean defining annotation, while it's not in CDI.
+     */
+    @BuildStep
+    BeanDefiningAnnotationBuildItem registerNamedAnnotation() {
+        return new BeanDefiningAnnotationBuildItem(CDI_NAMED_ANNOTATION);
```

Comment:
What's the default scope of these `@Named` beans in Spring?

Reply:
I assume Singleton? But I am not sure

Reply:
Yes, my question basically was: we need to check it and be consistent with what Spring does :).

Reply:
Yup 👍. I am 90% certain that's the case. Hopefully @rquinio can provide some info since I don't really have much computer access for a few days

Reply:
I confirm it defaults to singleton.
I'll put that explicitly and also allow those beans to be removable if never injected anywhere as it's not the default: 
`BeanDefiningAnnotationBuildItem(CDI_NAMED_ANNOTATION, CDI_SINGLETON_ANNOTATION, true)`

Reply:
Does Spring have this notion of removable beans? If not, I would keep them unremovable.

Reply:
Not that I know, I'll do that.